inttest/cli/cli_test.go

```diff
@@ -54,70 +56,96 @@ func (s *CliSuite) TestK0sCliCommandNegative() {
 
 func (s *CliSuite) TestK0sCliKubectlAndResetCommand() {
 	ssh, err := s.SSH(s.ControllerNode(0))
-	s.Require().NoError(err)
+	s.Require().NoError(err, "failed to SSH into controller")
 	defer ssh.Disconnect()
 
-	s.T().Log("running k0s install command")
-	_, err = ssh.ExecWithOutput("k0s install controller --enable-worker --disable-components konnectivity-server,metrics-server")
-	s.Require().NoError(err)
+	s.T().Run("preflightChecks", func(t *testing.T) {
+		out, err := ssh.ExecWithOutput(fmt.Sprintf(`
+			set -eu
+			fakeDir="$(mktemp -d)"
+			trap "rm -rf -- '$fakeDir'" INT EXIT
+			printf '#!/bin/sh\necho spoof' >"$fakeDir/uname"
```

Comment:
If this works it means that we depend on an external `uname` binary instead of using the [uname(2) syscall](https://man7.org/linux/man-pages/man2/uname.2.html), and I don't think we want depend on any external binaries.

Yup, k8s/system-validators execs external `uname`:
https://github.com/kubernetes/system-validators/blob/73d9d3d9f43503b1a64ec0506080252c683f8408/validators/os_validator.go#L40